odxtools/singleecujob.py

```diff
@@ -163,6 +258,72 @@ def __post_init__(self) -> None:
         if not self.neg_output_params:
             self.neg_output_params = NamedItemList(short_name_as_id, [])
 
+    @staticmethod
+    def from_et(et_element, doc_frags: List[OdxDocFragment]) \
```

Comment:
for the first argument? yes, but if this ought to be done for all `_from_et` functions, making mypy happy again will lead to another giant patch. This will be one of my next projects...